### DISCUSSION TITLE
Allow more editing

### DISCUSSION THREAD

**Discussion by hadielmentor** [woocommerce/woocommerce#52069] [Checkout Flow]
You have created a very attractive theme and mode, but there is little styling, the buttons don't get color, you can't add personal fields, you can't change the names of the fields or even remove them as necessary or unnecessary. Thank you for considering these points in the new water date. It is very great

**Comment by alexflorisca** [comment]
Thanks for your comment @hadielmentor. Which "theme and mode" are you referring to? If you could be a bit more specific with your feedback we can make sure to address what we can.
